src/benchmarks/real-world/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks/Harness/Config.cs

```diff
@@ -0,0 +1,53 @@
+﻿using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using Microsoft.ML.Benchmarks.Harness;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class RecommendedConfig : ManualConfig
+    {
+        public RecommendedConfig()
+        {
+            Add(DefaultConfig.Instance); // this config contains all of the basic settings (exporters, columns etc)
+
+            Add(GetJobDefinition() // job defines how many times given benchmark should be executed
+                .WithCustomBuildConfiguration(GetBuildConfigurationName())
+                .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark
```

Comment:
ML.NET repo required that (so we could reference ever native dependency in explicit way) but we don't need it here (it will be provided when we just install ML.NET NuGet package)
```suggestion
```